crates/rspack_plugin_mf/src/container/embed_federation_runtime_plugin.rs

```diff
@@ -0,0 +1,244 @@
+//! # EmbedFederationRuntimePlugin
```

Comment:
Embeds the module federation runtime module into runtime chunks, handles management of startup calls and rendering the startup of entrypoints.